Yoast/wordpress-seo
# Use loader icon

## Summary

Originally intended to address the score that changes during the initialisation of a category page.

The issue is that, when opening a category-edit page. The overall SEO score in the metabox is set multiple times. Once it's set to undefined, at this point the overall score is still unknown. Once it's calculated wrongly, giving a wrong score as a result. Only the final score is correct. 

The intention of this PR was originally to try and remove the wrong score that is set in the store at some point in time. That goal hasn't been reached yet.

What this PR does do is that, during the time when the SEO/Readability score is unknown, it replaces the grey analysis result with a spinner showing that it's still being loaded.

This PR can be summarized in the following changelog entry:

* Changes the overall analysis scores to a loader icon when the analysis results are still being calculated. 

## Relevant technical choices:

* Used the same CSS approach as used by the loader.

## Test instructions

This PR can be tested by following these steps:

* Checkout this branch and its sibling PR: https://github.com/Yoast/yoast-components/pull/709
* In the sibling pr: `yarn link`
* on this branch: `yarn && yarn link yoast-components && grunt && yarn start`
* Open a post-edit and/or term-edit page
* Check if, while the analysis is still being loaded, the icon appears as a spinner.


partially fixes: https://github.com/Yoast/YoastSEO.js/issues/1688 

## Quality assurance


Fixes https://github.com/Yoast/wordpress-seo/issues/10612
